sanitiser/_private.js

```diff
@@ -0,0 +1,31 @@
+var isObject = require('is-object');
+var isTruthy = require('./_truthy');
+
+// validate inputs, convert types and apply defaults
+function sanitize( req, default_value ){
+
+  req.clean = req.clean || {};
+  var params= req.query;
+
+  if (default_value === undefined) {
+    default_value = true;
+  }
+
+  default_value = !!default_value;
+
+  // ensure the input params are a valid object
+  if( !isObject( params ) ){
+    params = {};
+  }
+
+  if (params.private === undefined) {
```

Comment:
try to stick with hasOwnProperty function for checking if something is undefined:

```
if ( params.hasOwnProperty('private') ) {
  // defined
} else {
  // undefined
}